dist/livereload.js

```diff
@@ -1095,6 +1127,9 @@
   });
 
   LiveReload.on('connect', function() {
+    if (!!/true|1$/.test(LiveReload.options.animate)) {
```

Comment:
I think it's cleaner to put all parsing in `Options.set`, and have sanitized data inside the fields. (The same can be said about my own code, though. Maybe it's a separate change.)